scripts/backend_tests.py

```diff
@@ -249,7 +249,17 @@ def main():
         raise Exception('The delimiter in test_target should be a dot (.)')
 
     if parsed_args.test_target:
-        all_test_targets = [parsed_args.test_target]
+        if parsed_args.test_target.endswith('_test'):
+            all_test_targets = [parsed_args.test_target]
+        else:
+            print ''
+            print '.....................................................'
+            print 'WARNING : test_target flag should point to the test file.'
+            print '.....................................................'
```

Comment:
Just a confirmation, do we normally use . or -?

Reply:
Hmm..I think `-`. Nice catch, @nithusha21! @anubhavsinha98, can you please change this? Thanks!

Reply:
Yea I think we use `-` @anubhavsinha98 Would you change it please?

Reply:
Oops -- sorry @apb7 you already mentioned it.

Reply:
Okay, will change it !
Sorry my fault.